documentation/src/main/asciidoc/reference/search-dsl-predicate.asciidoc

```diff
@@ -784,11 +784,78 @@ but can be <<search-dsl-predicate-common-boost,boosted>>,
 either on a per-field basis with a call to `.boost(...)` just after `.field(...)`/`.fields(...)`
 or for the whole predicate with a call to `.boost(...)` after `.matchingAny(...)` or `.matchingAll(...)`.
 
+[[search-dsl-predicate-and]]
+// TODO YB: find references to _combining_queries and add references to _anding_queries
+== [[_anding_queries]] `and`: combine predicates as a conjunction
+
+The `and` predicate matches documents that match a conjunction of inner predicates, called "clauses".
+`and` clauses are required to match: if they don't match, then the `and` predicate will not match.
+
+Matching "and" clauses are taken into account during <<search-dsl-predicate-common-score,score>> computation.
+
+.Matching a document that matches all the multiple given predicates (~`AND` operator)
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java[tags=and]
+----
+<1> The hits *must* have a `title` field matching the text `robot`,
+independently of other clauses in the same boolean predicate.
+<2> The hits *must* have a `description` field matching the text `crime`,
+independently of other clauses in the same boolean predicate.
+<3> All returned hits will match *all* the clauses above:
+they will have a `title` field matching the text `robot`
+*and* they will have a `description` field matching the text `crime`.
+====
+
+[[search-dsl-predicate-and-other]]
+=== Options
+
+* The score of an `and` predicate is variable by default,
+but can be <<search-dsl-predicate-common-constantScore,made constant with `.constantScore()`>>.
+* The score of an `and` predicate can be <<search-dsl-predicate-common-boost,boosted>>
+with a call to `.boost(...)`.
+
+[[search-dsl-predicate-or]]
+// TODO YB: find references to _combining_queries and add references to _oring_queries
+== [[_oring_queries]] `or`: combine predicates as a disjunction
```

Comment:
Same comment about `_oring_queries`: where does that come from?

Reply:
Ditto!